x-pack/plugins/security_solution/server/lib/detection_engine/rule_types/eql/build_alert_group_from_sequence.ts

```diff
@@ -47,7 +47,8 @@ export const buildAlertGroupFromSequence = (
   buildReasonMessage: BuildReasonMessage,
   indicesToQuery: string[],
   alertTimestampOverride: Date | undefined,
-  publicBaseUrl?: string
+  publicBaseUrl?: string,
+  intendedTimestamp?: Date
```

Comment:
```suggestion
  intendedTimestamp: Date | undefined
```

Reply:
Fixed, other places, but here we will have error:
`A required parameter cannot follow an optional parameter`

Because parameters here is not an object